rbd/system/cli/rbd_cli_import_export_diff.py

```diff
@@ -41,16 +42,11 @@ def exec_cmd(args):
     if k_m:
         combinations = cli.generate_combinations('image_size', 'object_size',
                                                  'image_format', 'data_pool')
-        invalid = [val for val in combinations
-                   if (cli.search_param_val('--image-format', val) != 0 and
-                       cli.search_param_val('--image-format', val)
```

Comment:
@Gopi-Patta what was the problem here wrt nautilus? any rbd cli changes

Reply:
@Gopi-Patta ??

Reply:
@XDsubhash  removed as image-format 1 is not supporting now.

Reply:
@Gopi-Patta what abt in luminous ? image-format is working fine right?

Reply:
@Gopi-Patta , I accidentally merged this PR. please check on luminous as well. there is no separate branch for luminous as well.

Reply:
@XDsubhash Image-format 1 will work in luminous with warning. Please ignore this PR as i will send one more by including image-format 1 when it is luminous.